Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "simple-kbs"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio = { version = "1.0", features = ["full"] }
+tonic = "0.5"
+prost = "0.8"
+anyhow = "1.0"
+uuid = { version = "0.8.2", features = ["serde", "v4"] }
+sev = { version = "0.2.0", features = ["openssl"] }
+codicon = "3.0"
+bincode = "1.3.3"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+base64 = "0.13.0"
+lazy_static = "1.4.0"
+clap = "3.0.13"
+mysql = "22.0.0"
+log = "0.4.14"
+env_logger = "0.9.0"
```

Comment:
After merging to main, we can try (in a separate PR) to upgrade these: for example, `uuid` has released 1.1.1, `clap` released 3.1.18, as well as tokio and tonic, and maybe others.

Specifically `uuid` added the `to_bytes_le()` method (https://github.com/uuid-rs/uuid/pull/599) which can replace my ugly implementation at the top of `request.rs`.